<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScorePage</class>
 <widget class="QWidget" name="ScorePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <item>
    <widget class="QLabel" name="title_label">
      <property name="text">
        <string>Schedule &amp; Scoring</string>
      </property>
      <property name="alignment">
        <set>Qt::AlignCenter</set>
      </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="controlsLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="update_button">
       <property name="text">
        <string>Update Rankings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_button">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="csv_button">
       <property name="text">
        <string>Export CSV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pdf_button">
       <property name="text">
        <string>Export PDF</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="controlsSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="auto_checkbox">
       <property name="text">
        <string>Auto-update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="analytics_toggle">
       <property name="text">
        <string>Show analytics</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="layout_stack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="desktop_container">
      <layout class="QVBoxLayout" name="desktopLayout">
       <property name="contentsMargins">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>0</width>
         <height>0</height>
        </rect>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QScrollArea" name="scroll_area">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="matches_host">
           <layout class="QVBoxLayout" name="matches_layout">
            <property name="spacing">
             <number>12</number>
            </property>
            <property name="contentsMargins">
             <rect>
              <x>24</x>
              <y>16</y>
              <width>24</width>
              <height>32</height>
             </rect>
            </property>
           </layout>
          </widget>
         </widget>
         <widget class="QFrame" name="summary_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QVBoxLayout" name="summaryLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="contentsMargins">
            <rect>
             <x>12</x>
             <y>12</y>
             <width>12</width>
             <height>24</height>
            </rect>
           </property>
           <item>
            <widget class="QStackedWidget" name="summary_stack">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="analytics_page">
              <layout class="QVBoxLayout" name="analyticsLayout">
               <item>
                <widget class="QTabWidget" name="analytics_tabs">
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="rankings_tab">
                  <attribute name="title">
                   <string>Rankings</string>
                  </attribute>
                  <layout class="QVBoxLayout" name="rankingsLayout">
                   <item>
                    <widget class="QWidget" name="filter_widget">
                     <layout class="QHBoxLayout" name="filterLayout">
                      <property name="contentsMargins">
                       <rect>
                        <x>0</x>
                        <y>5</y>
                        <width>0</width>
                        <height>5</height>
                       </rect>
                      </property>
                      <property name="spacing">
                       <number>8</number>
                      </property>
                      <item>
                       <widget class="QLabel" name="filter_label">
                        <property name="text">
                         <string>Filter:</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="search_input">
                        <property name="placeholderText">
                         <string>Search playerâ€¦</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QSpinBox" name="top_spin">
                        <property name="specialValueText">
                         <string>All players</string>
                        </property>
                        <property name="prefix">
                         <string>Top </string>
                        </property>
                        <property name="minimum">
                         <number>0</number>
                        </property>
                        <property name="maximum">
                         <number>999</number>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QTableWidget" name="rank_table">
                     <property name="columnCount">
                      <number>5</number>
                     </property>
                     <property name="alternatingRowColors">
                      <bool>true</bool>
                     </property>
                     <property name="horizontalScrollBarPolicy">
                      <enum>Qt::ScrollBarAsNeeded</enum>
                     </property>
                     <property name="verticalScrollBarPolicy">
                      <enum>Qt::ScrollBarAsNeeded</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="plot_tab">
                  <attribute name="title">
                   <string>Performance chart</string>
                  </attribute>
                  <layout class="QGridLayout" name="plotLayout">
                   <property name="horizontalSpacing">
                    <number>16</number>
                   </property>
                   <property name="verticalSpacing">
                    <number>12</number>
                   </property>
                   <property name="contentsMargins">
                    <rect>
                     <x>16</x>
                     <y>12</y>
                     <width>16</width>
                     <height>12</height>
                    </rect>
                   </property>
                   <item row="0" column="0" colspan="2">
                    <widget class="QLabel" name="plot_label">
                     <property name="text">
                      <string>Graph players</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QListWidget" name="player_list"/>
                   </item>
                   <item row="1" column="1">
                    <widget class="QWidget" name="plot_placeholder">
                     <layout class="QVBoxLayout" name="plotPlaceholderLayout">
                      <property name="contentsMargins">
                       <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>0</width>
                        <height>0</height>
                       </rect>
                      </property>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="analytics_cover">
              <layout class="QVBoxLayout" name="coverLayout">
               <property name="contentsMargins">
                <rect>
                 <x>24</x>
                 <y>24</y>
                 <width>24</width>
                 <height>24</height>
                </rect>
               </property>
               <item>
                <spacer name="coverTopSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="cover_label">
                 <property name="text">
                  <string>Analytics are hidden</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="cover_hint">
                 <property name="text">
                  <string>Use &apos;Show analytics&apos; to reveal rankings and charts again.</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="coverBottomSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mobile_container">
      <layout class="QVBoxLayout" name="mobile_layout">
       <property name="contentsMargins">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>0</width>
         <height>0</height>
        </rect>
       </property>
       <property name="spacing">
        <number>12</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="back_button">
     <property name="text">
      <string>Back to Setup</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
